<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*登录按钮样式*/
#Sure_Button{
	border-radius:4px;
	border:none;
	background-color: #09A3DC;
	color:#fff;
}

/*登录按钮鼠标悬停样式*/
#Sure_Button:hover{
	background-color:rgb(60,195,245);
}
/*输入框悬停样式*/
#RealName_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#UserName_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#PassWord_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#PassWordTwo_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#Place_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#PhoneNumber_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
#IdCard_Edit:hover{
	
	border:1px solid rgb(21,131,221)
	
}
/*用户名输入框样式*/
#RealName_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#UserName_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#PassWord_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#PassWordTwo_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#Place_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#PhoneNumber_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}
#IdCard_Edit{
	border-top-left-radius:4px;
	border-top-right-radius:4px;
	border-bottom-right-radius:4px;
	border-bottom-left-radius:4px;	
	border:1px solid rgb(209,209,209);
}



</string>
  </property>
  <widget class="QLabel" name="UserName_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassWord_Label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="PassWordTwo_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="Place_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>常驻地</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNumber_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>手机号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="RealName_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>52</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>真实姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="IdCard_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Songti SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>身份证号码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserName_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWord_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassWordTwo_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Place_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RealName_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumber_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>320</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdCard_Edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Sure_Button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Songti SC&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="Backgroud_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/BackgroudTwo.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Backgroud_Label</zorder>
  <zorder>UserName_Label</zorder>
  <zorder>PassWord_Label</zorder>
  <zorder>PassWordTwo_Label</zorder>
  <zorder>Place_Label</zorder>
  <zorder>PhoneNumber_Label</zorder>
  <zorder>RealName_Label</zorder>
  <zorder>IdCard_Label</zorder>
  <zorder>UserName_Edit</zorder>
  <zorder>PassWord_Edit</zorder>
  <zorder>PassWordTwo_Edit</zorder>
  <zorder>Place_Edit</zorder>
  <zorder>RealName_Edit</zorder>
  <zorder>PhoneNumber_Edit</zorder>
  <zorder>IdCard_Edit</zorder>
  <zorder>Sure_Button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
